layer/Python/Dockerfile

```
@@ -1,19 +1,28 @@
 FROM public.ecr.aws/lambda/python:3.8
 
-
-
 ARG PACKAGE_SUFFIX=''
 
 USER root
 WORKDIR /tmp
 
-
-# PACKAGE_SUFFIX = '[pydantic]==1.23.0'
-# PACKAGE_SUFFIX = '[pydantic]'
-# PACKAGE_SUFFIX = '=='1.23.0'
+# PACKAGE_SUFFIX = '[all]==2.0.0'
+# PACKAGE_SUFFIX = '[all]'
+# PACKAGE_SUFFIX = '=='2.0.0'
 # PACKAGE_SUFFIX = ''
 
-
-RUN yum update -y && yum install -y zip unzip wget tar gzip
-
-RUN pip install -t /asset/python aws-lambda-powertools$PACKAGE_SUFFIX
\ No newline at end of file
+RUN yum update -y && yum install -y zip unzip wget tar gzip binutils
+
+RUN pip install -t /asset/python aws-lambda-powertools$PACKAGE_SUFFIX
+
+# Removing nonessential files
+RUN cd /asset && \
+  # remove boto3 and botocore (already available in Lambda Runtime)
+  rm -rf python/boto* && \
+  # remove boto3 dependencies
+  rm -rf python/s3transfer* python/*dateutil* python/urllib3* && \
```

```suggestion
  rm -rf python/s3transfer* python/*dateutil* python/urllib3* python/six* && \
```

Adding six, committing and merging. `six` is a library to support compat between Python 2 and 3.